### ISSUE TITLE
GUI: Applying Algorithm Parameters to Multiple Channels

### ISSUE THREAD

**Issue by ykukkim** [GallVp/emgGO#3]
I am trying to run your code under for loop with the appropriate parameter settings.

The detection works smoothly, which is amazing. However, I have a question regards to running the code with for loop.

The for loop will not go through to the next section unless I close the figure manually, is there a way for me to change this setting? 

I would like to change this setting so that it goes through every data with 14 channels, figures being saved and closed so that the next data can be processed.

If this is possible, let me know.

Thank you,

**Comment by GallVp** [comment]
I assume you are referring to `emgEventsDetectTool` function. The GUI halts the further execution of this function by invoking MATLAB's `wait` function at line [223](https://github.com/GallVp/emgGO/blob/823a089ce30dd1c540f550ebc957a9911f5058b6/emgEventsDetectTool.m#L223). Thus, if you comment out this line, the function will open the GUI and return immediately. However, it will not close the GUI. To do that and to return the processed data, you will have to add `closeFigure([], [])` in the place of `wait(H)` at line 223.

Cheers
Usman R.

**Comment by ykukkim** [comment]
Thank you Usman.

Also, I have been wondering about your parameter vectors. 

Please correct me if I am wrong, but the parameter vector seems to be consistent throughout all channels. However, if EMG sensors are attached to different muscle groups, shouldn't the parameter vectors also be set accodringly to the muscle group as they all behave differently.

I would be interested to know your thoughts on this.

Thank you,

Kind regards,

Yong Kim

**Comment by GallVp** [comment]
The parameters are initialised to same values across the channels. However, in the GUIs, each channel has its own separate set of parameters. You can also initialise with different parameters by passing in a matrix of parameters with the `options.subParams`. This logic can be see [here](https://github.com/GallVp/emgGO/blob/5676717f75ed260bdb241221e026d628a3227632/emgEventsDetectTool.m#L92).

**Comment by ykukkim** [comment]
Thank you Usman, 

I am quite curious about your "AutoTune" function. I am trying to utilise this function but I am not too sure if I understood this correctly.

I should apply this function after Onset/offsets have been adjusted either by manually or using the "AutoFind". However, if I adjusted Onsets/offsets using the function, the corresponding parameters are already updated. Therefore, if I adjusted the dataset manually which does not update the parameters so I should then use this function to apply the parameter settings shoudln't I?
